web-ng/src/app/services/auth/auth.service.ts

```diff
@@ -49,7 +50,9 @@ export class AuthService {
           return of(null);
         }
       }),
-      map(user => user || ANONYMOUS_USER),
+      map(
+        user => (user && { ...user, isAuthenticated: true }) || ANONYMOUS_USER
```

Comment:
Rather than rebuild the user here, just set `isAuthenticated` when building the user in `FirebaseDataConverter`.